src/App.vue

```diff
@@ -10,15 +10,9 @@ const steps = 5;
 const currentStep = ref(0);
```

Comment:
Don’t forget to correct this for the deleted steps

Reply:
Maybe find a new way to the progress bar if there’s only so little steps

Reply:
The data crunching still takes some time so I think the current implementation still makes some sense